recipes/stare/stare-activate.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+export STARE_INCLUDE_DIR=${CONDA_PREFIX}/include/STARE
+export STARE_LIB_DIR=${CONDA_PREFIX}/lib
```

Comment:
Are these needed? Ideally the code should be compiled to default to these values (they can be hard coded in the binary and conda will fix the to the correct path at install time)

Reply:
Other packages (e.g. on PyPi) currently use these during their build. Is there a better approach?

Reply:
Ah you're wanting these for downstream dependencies to be able to build against? If so you probably don't need to do anything when using conda-forge's compilers as these are the default locations.

Reply:
I've removed these. Since we're in a conda-forge environment, they should be easier to find.